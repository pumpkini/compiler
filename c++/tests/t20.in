true2 + 1 == 2