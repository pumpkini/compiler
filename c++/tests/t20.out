T_ID true2
+
T_INTLITERAL 1
==
T_INTLITERAL 2